docs/release-notes/1.5.3/release-1.5.3.md

```diff
@@ -0,0 +1,41 @@
+- Update OnnxRuntime to 1.6 ([#5529](https://github.com/dotnet/machinelearning/pull/5529))
+- Updated tensorflow.net to 0.20.0 ([#5404](https://github.com/dotnet/machinelearning/pull/5404))
+
+## **Bug Fixes**
+- **AutoML.NET specific fixes**.
+  - Fixed AutoFitMaxExperimentTimeTest ([#5506](https://github.com/dotnet/machinelearning/pull/5506))
+  - Fixed code generator tests failure ([#5520](https://github.com/dotnet/machinelearning/pull/5520))
+  - Use Timer and ctx.CancelExecution() to fix AutoML max-time experiment bug ([#5445](https://github.com/dotnet/machinelearning/pull/5445))
+  - Handled exception during GetNextPipeline for AutoML ([#5455](https://github.com/dotnet/machinelearning/pull/5455))
+  - Fixed issue when parsing float string fails on pl-PL culture set using Regression Experiment ([#5163](https://github.com/dotnet/machinelearning/pull/5163))
+  - Fixed MaxModels exit criteria for AutoML unit test ([#5471](https://github.com/dotnet/machinelearning/pull/5471))
+- Fixed memory leaks from OnnxTransformer ([#5518](https://github.com/dotnet/machinelearning/pull/5518))
+- Fixed memory leak in object pool ([#5521](https://github.com/dotnet/machinelearning/pull/5521))
+- Fixed Onnx Export for ProduceWordBags ([#5435](https://github.com/dotnet/machinelearning/pull/5435))
+- Upgraded boundary calculation and expected value calculation in SrCnnEntireAnomalyDetector ([#5436](https://github.com/dotnet/machinelearning/pull/5436))
+- Fixed SR anomaly score calculation at beginning ([#5502](https://github.com/dotnet/machinelearning/pull/5502))
+- Improved error message in ColumnConcatenatingEstimator ([#5444](https://github.com/dotnet/machinelearning/pull/5444))
+- Fixed issue 5020, allow ML.NET to load tf model with primitive input and output column ([#5468](https://github.com/dotnet/machinelearning/pull/5468))
+- Fixed issue 4322, enable lda summary output ([#5260](https://github.com/dotnet/machinelearning/pull/5260))
+- Fixed perf regression in ShuffleRows ([#5417](https://github.com/dotnet/machinelearning/pull/5417))
+- Change the _maxCalibrationExamples default on CalibratorUtils ([#5415](https://github.com/dotnet/machinelearning/pull/5415))
+
+
+## **Build / Test updates**
+- Migrated to [Arcade](https://github.com/dotnet/arcade/) build system that is used my multiple dotnet projects. This will give increased build/CI efficiencies going forward. Updated build instructions can be found in the docs/building folder
+- Fixed MacOS builds ([#5467](https://github.com/dotnet/machinelearning/pull/5467) and [#5457](https://github.com/dotnet/machinelearning/pull/5457))
+
+## **Documentation Updates**
+- Fixed Spelling on stopwords ([#5524](https://github.com/dotnet/machinelearning/pull/5524))
+- Changed LoadRawImages Sample ([#5460](https://github.com/dotnet/machinelearning/pull/5460))
+
+
+## **Breaking Changes**
+- None
```

Comment:
Could include a contributors/thanks section for any one submitting a PR; especially important for community contributions.

Reply:
I have included a thanks next to the PR in the next iteration.

---
In reply to: [539740764](https://github.com/dotnet/machinelearning/pull/5543#discussion_r539740764) [](ancestors = 539740764)